src/components/Common/Fields/SelectField/SelectField.stories.tsx

```diff
@@ -0,0 +1,202 @@
+import type { Story } from '@ladle/react'
+import { FormProvider, useForm } from 'react-hook-form'
+import { useEffect } from 'react'
+import { SelectField } from './SelectField'
+
+interface FormWrapperProps {
+  children: React.ReactNode
+  defaultValues?: {
+    category?: string
+    priority?: string
+    status?: string
+  }
+}
+
+const FormWrapper = ({ children, defaultValues = {} }: FormWrapperProps) => {
+  const methods = useForm({
+    defaultValues: {
+      category: defaultValues.category || '',
+      priority: defaultValues.priority || '',
+      status: defaultValues.status || '',
+    },
+    mode: 'onTouched',
+  })
+
+  // Log form state changes to console
```

Comment:
You might be able to do a similar thing with ladle actions?
https://ladle.dev/docs/actions